### PR TITLE
add new subsections for spurious timeout and spurious loss by ack

### LINKED ISSUE #90: Citing Experimental RFCs as if being a part of CUBIC
Markku Kojo said,
> The draft says that CUBIC MAY implement DSACK [RFC3708], limited slow
  start [RFC 3742], [RFC7661] and hybrid slow start [cites a paper].
  Aren't the first three down references? Not sure if it is appropriate
  for a Stds Track document to cite experimental work or a paper like
  this even though it's a MAY.

### PR DESCRIPTION
**Pull request by goelvidhi**
Closes #90 

This is an attempt to resolve the conflicting discussion regarding response to spurious events. Hopefully, we can reach some conclusion in this PR. :-)

### CODE REVIEW

**Review by larseggert — APPROVED** [review]

**Comment by goelvidhi** [comment]
@nealcardwell @lisongxu @Injongrhee @markkukojo could you please review this PR? It would be good to conclude this before the next IETF. Thank you. :-)

**Comment by lisongxu** [comment]
@goelvidhi Thanks, Vidhi!  A reader may be confused with " When a spurious timeout is detected, the sender MAY restore the congestion control state and adapt the retransmission timer to avoid further spurious timeouts" at line 759, as we have not described how to save the congestion control state yet (line 782). We move this sentence to line 800? Thanks

**Comment by goelvidhi** [comment]
> @goelvidhi Thanks, Vidhi! A reader may be confused with " When a spurious timeout is detected, the sender MAY restore the congestion control state and adapt the retransmission timer to avoid further spurious timeouts" at line 759, as we have not described how to save the congestion control state yet (line 782). We move this sentence to line 800? Thanks

@lisongxu, as discussed in the issue, we want to recommend RFC 4015's response algorithm for spurious timeouts. Only for fast retransmits, we are recommending the algo we described in Section 4.9.2. I have reworded the text a little bit to clarify things a bit. Please take a look. Thanks.

**Review by lisongxu — APPROVED** [review]
Thanks

**Comment by goelvidhi** [comment]
@nealcardwell @markkukojo could you please review these changes? It would be great to get to reach closure on this.

**Comment by larseggert** [comment]
Could we finalize this PR?

File: draft-ietf-tcpm-rfc8312bis.md [hunk 1]
```diff
@@ -748,9 +748,39 @@ of packet reordering could cause multiple congestion window reduction
...
-the current value of the following variables before the congestion
-window reduction.
+For TCP, there are two types of spurious events - spurious timeouts
+and spurious fast retransmits. In case of QUIC, there are no spurious
+timeouts as the loss is only detected after receiving an ACK.
+
+### Spurious timeout
+
+An implementation MAY detect spurious timeouts based on the mechanisms
+described in Forward RTO-Recovery {{!RFC5682}}. Experimental alternative
```
Comment (nealcardwell) [inline draft-ietf-tcpm-rfc8312bis.md:758]: alternative -> alternatives

(Or: "One experimental alternative is Eifel ...")

File: draft-ietf-tcpm-rfc8312bis.md [hunk 2]
```diff
@@ -764,12 +794,10 @@ prior\_W\_{est} = W_{est} \\
...
 {: artwork-align="center" }
 
-CUBIC MAY implement an algorithm to detect spurious retransmissions,
-such as Forward RTO-Recovery {{!RFC5682}}. Experimental alternatives
-include DSACK {{?RFC3708}} and Eifel {{?RFC3522}}. Once a spurious
-congestion event is detected, CUBIC SHOULD restore the original values
-of above-mentioned variables as follows if the current *cwnd* is lower
-than *prior_cwnd*. Restoring the original values ensures that CUBIC's
+Once the previously declared packet loss is confirmed to be spurious,
+CUBIC MAY restore the original values of above-mentioned variables
```
Comment (nealcardwell) [inline draft-ietf-tcpm-rfc8312bis.md:798]: I'd suggest:

  the original values of the above-mentioned variables

**Review by nealcardwell — APPROVED** [review]
The substance looks good to me. Made some wordsmithing suggestions.

File: draft-ietf-tcpm-rfc8312bis.md (hunk 1 above)
Comment (larseggert) [inline draft-ietf-tcpm-rfc8312bis.md:758]:
```suggestion
described in Forward RTO-Recovery {{!RFC5682}}. Experimental alternatives
```

File: draft-ietf-tcpm-rfc8312bis.md (hunk 2 above)
Comment (larseggert) [inline draft-ietf-tcpm-rfc8312bis.md:798]:
```suggestion
CUBIC MAY restore the original values of the above-mentioned variables
```

**Pull request merged**